app/components/Footer.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Footer = () => (
+  <div className="footer text-center">
+    <div className="vcenter muted">
+      <span>
+        <span className="glyphicon glyphicon-wrench" /> <span className="glyphicon glyphicon-heart" /> don'tclickme :) <a href="http://briansultimatetraining.blogspot.com/">uhhhhh</a> <span className="glyphicon glyphicon-education" />
```

Comment:
This line seems a little long. Might be worth splitting across a couple other lines.